docs/reference/buildx_build.md

```diff
@@ -271,6 +271,148 @@ $ docker buildx build --cache-from=type=s3,region=eu-west-1,bucket=mybucket .
 
 More info about cache exporters and available attributes: https://github.com/moby/buildkit#export-cache
 
+### <a name="call"></a> Invoke a subrequest (--call)
```

Comment:
```suggestion
### <a name="call"></a> Invoke a frontend method (--call)
```

I realized we probably want to name these methods. I defaulted to subrequests because of `subrequests.describe`.